test/notify.sh

```diff
@@ -0,0 +1,58 @@
+#!/bin/sh
+set -x
+
+if [ x"${srcdir}" = x ]; then
+    srcdir=.
+fi
+. ${srcdir}/lib.sh
+
+NOT1=${DIR}/${NM}-1.sh
+NOT1STAMP=${DIR}/${NM}-1.stamp
+NOT2=${DIR}/${NM}-2.sh
+NOT2STAMP=${DIR}/${NM}-2.stamp
+
+printf '#!/bin/sh -\necho script 1: $* > '${NOT1STAMP}'\n' > ${NOT1}
+printf '#!/bin/sh -\necho script 2: $* > '${NOT2STAMP}'\n' > ${NOT2}
+chmod 0755 ${NOT1} ${NOT2}
+
+cat <<EOF > ${CONF}
+notify ${NOT1}
+# Match all log messages, store in RC5424 format and rotate every 1 KiB
+*.*       -${LOG}    ;rotate=1k:2,RFC5424
+notify ${NOT2}
+EOF
+
+../src/syslogd -m1 -b :${PORT2} -d -sF -f ${CONF} -p ${SOCK2} -p ${ALTSOCK} -P ${PID2} >${LOG2} &
+sleep 3
+cat ${PID2} >> "$DIR/PIDs"
```

Comment:
Like I mentioned in the general comment, I'm planning a minor refactor of lib.sh so we can add generic arguments to the setup() function instead, meaning we don't need to start a second syslogd like you do here.  I'll see what I can do about that already later tonight.

Reply:
Did that. Much easier now to do this, thanks for your work last weekend!